.github/workflows/generator_generic_slsa3.yml

```diff
@@ -97,6 +97,9 @@ on:
       provenance-name:
         description: "The artifact name of the signed provenance. (A file with the intoto.jsonl extension)."
         value: ${{ jobs.generator.outputs.provenance-name }}
+      provenance-uuid:
+        description: "You could also use the Rekor Search UI: https://search.sigstore.dev/?uuid= to view the log entry details."
```

Comment:
nit: Describe what the output is: `The rekor UUID ... It may be used to search ...`

Also, don't forget to update the README https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/generic#workflow-outputs

Reply:
Thank you for the feedback. I have made the requested changes